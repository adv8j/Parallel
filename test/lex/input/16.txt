@TaskGroup(log="file.txt)
{
    // " // this gives error
}