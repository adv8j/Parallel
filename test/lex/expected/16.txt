TASKGROUP
LPAREN
LOG
ASSIGN
STRING_LITERAL
<ERROR?>
LBRACE
single line comment
RBRACE
